server/data/helpers/classesHelper.js

```diff
@@ -87,6 +87,11 @@ module.exports = {
     );
   },
 
+  getTeacherClasses: async teacher_id =>
+    db('teachers_classes_refreshrs as tcr')
+      .join('classes as c', 'c.id', 'tcr.class_id')
+      .where({ teacher_id }),
+
```

Comment:
Oh cool so this gets the teacher's classes and adds the student to the class at once, very nice!